client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Continue,
+	Tour,
+	Step,
+	ButtonRow,
+	Next,
+	Quit,
+	Link,
+} from 'layout/guided-tours/config-elements';
+import { hasSelectedSitePremiumOrBusinessPlan } from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const SimplePaymentsEndOfYearGuide = makeTour(
+	<Tour
+		name="simplePaymentsEndOfYearGuide"
+		version="20171103"
+		path={ [
+			'/stats',
+			'/plans',
+			'/pages',
+			'/posts',
+			'/media',
+			'/comments',
+			'/types',
+			'/themes',
+			'/sharing',
+			'/people',
+			'/plugins',
+			'/settings',
+		] }
+		when={ and( isDesktop, hasSelectedSitePremiumOrBusinessPlan ) }
+	>
+		<Step name="init" placement="right">
+			<p>
+				{ translate(
+					'Now is a great time to add a {{strong}}Payment Button!{{/strong}} Do you have something to sell? ' +
+						'Would you like to accept donations? Get your site ready for Cyber Monday and end-of-year giving. ' +
+						'People will spend $3 billion online!',
+					{
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<img
+				src="https://i0.wp.com/en-support.files.wordpress.com/2017/03/editor-content-area_360.gif"
+				style={ { marginBottom: '10px', border: '3px solid #00AADC', borderRadius: '4px' } }
+			/>
+			<ButtonRow>
+				<Next step="add-new-page">{ translate( 'Get started!' ) }</Next>
```

Comment:
Hi! I've found a possible matching string that has already been translated **89** times:
`translate( 'Get Started' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/guided-tour-end-of-year)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).